pandas/core/indexing.py

```diff
@@ -1993,7 +1993,14 @@ def _setitem_with_indexer_missing(self, indexer, value):
                     df = df.infer_objects()
                 self.obj._mgr = df._mgr
             else:
-                self.obj._mgr = self.obj.append(value)._mgr
+                from pandas.core.reshape.concat import concat
+
+                df = value.to_frame().T
```

Comment:
hmm this not great. cc @jbrockmendel can you suggest anything better here

Reply:
this looks very nearly cut/paste from DataFrame.append.  the only difference is that we actually have _tests_ for DataFrame.append.  If this code has to exist somewhere, then we might as well just implement DataFrame._append so we can adapt the existing tests (and im guessing the scope of DataFrame._append would be narrower so once the deprecation is enforced we could rip out a bunch of it)